.cursor/mcp.json

```
@@ -1,10 +1,10 @@
 {
   "mcpServers": {
-    "runpod": {
+    "runpod-test": {
```

```suggestion
    "runpod": {
```

Was `-test` an artifact of local testing or is this to imply the user may have the production and a test copy of the MCP server running?

---

It's the latter- I had both installed so renamed the dev version for clarity.